### DISCUSSION TITLE
Problems with babel and react-native-reanimated

### DISCUSSION THREAD

**Discussion by Emanuel-Cordeiro** [software-mansion/react-native-reanimated#5756] [Q&A]
I'm trying to build a react native application using react-navigation-drawer, which requires react-native-reanimated in order to work. Even if I follow de documentation I get this error while bundling my project:

error: index.js: Cannot find module 'react-native-reanimated/´plugin'
Require stack:
.\Aplicativo\node_modules\@babel\core\lib\config\files\plugins.js
.\Aplicativo\node_modules\@babel\core\lib\config\files\index.js
.\App Planilha\Aplicativo\node_modules\@babel\core\lib\index.js
.\App Planilha\Aplicativo\node_modules\metro-transform-worker\src\index.js
.\App Planilha\Aplicativo\node_modules\metro\src\DeltaBundler\Worker.flow.js
.\App Planilha\Aplicativo\node_modules\metro\src\DeltaBundler\Worker.js
.\App Planilha\Aplicativo\node_modules\jest-worker\build\workers\processChild.js

I reinstalled all dependencies and put the plugin in babel.config.js, just like the docs, and cleaned the application cache, but it didnt work. Using react-navigation-stack or tabs works just fine.

There is a sample of the project in https://github.com/Emanuel-Cordeiro/Aplicativo.

I'm using android studio as an emulator.

**Comment by MatiPl01** [comment]
Hey!

Have you managed to fix the issue? I am seeing multiple discussions where people complain about the drawer navigation and I think that something might be wrong with this library.

The link to your sample project no longer works so I cannot test it locally.
